core/services/relay/evm/chain_writer.go

```diff
@@ -30,11 +33,13 @@ type ChainWriterService interface {
 // Compile-time assertion that chainWriter implements the ChainWriterService interface.
 var _ ChainWriterService = (*chainWriter)(nil)
 
-func NewChainWriterService(logger logger.Logger, client evmclient.Client, txm evmtxmgr.TxManager, config types.ChainWriterConfig) (ChainWriterService, error) {
+func NewChainWriterService(logger logger.Logger, client evmclient.Client, txm evmtxmgr.TxManager, estimator gas.EvmFeeEstimator, config types.ChainWriterConfig) (ChainWriterService, error) {
 	w := chainWriter{
-		logger: logger,
-		client: client,
-		txm:    txm,
+		logger:      logger,
+		client:      client,
+		txm:         txm,
+		ge:          estimator,
+		maxGasPrice: config.MaxGasPrice,
```

Comment:
would it make sense to have a default value?

Reply:
I don't think that would be necessary since this is already a required config in the gas estimator https://github.com/smartcontractkit/chainlink/blob/4fdfffdb4c4a6f16fcdb388fe38642398ce130da/core/chains/evm/config/config.go#L134

Reply:
Added a default of 500 GWEI